### PR TITLE
Porting code to Python 3

### LINKED ISSUE #12: Please port to Python 3

### PR DESCRIPTION
**Pull request by sergeybe**
It closes #12.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -2,13 +2,14 @@
...
-      version='0.0.1',
-      description='FFX',
-      author='Kevin P. Dyer',
-      author_email='kpdyer@gmail.com',
-      url='https://github.com/kpdyer/libffx',
-      packages=['ffx'],
-      )
+setup(
+    name='ffx',
+    install_requires=['gmpy', 'pycrypto', 'six'],
```
Comment (joaonc) [inline setup.py]: You might want to use `pycryptodome` instead, which is a (mostly) drop replacement to `pycrypto`, which is now defunct.

File: .travis.yml [hunk 2]
```diff
@@ -2,12 +2,14 @@ language:
...
   - gcc
-  - clang 
+  - clang
 before_install:
   - sudo apt-get update -qq
   - sudo apt-get install -y libgmp-dev
 python:
   - "2.7"
+  - "3.5"
+  - "3.6"
```
Comment (joaonc) [inline .travis.yml]: Any idea why Python 3.7 caused Travis tests to fail?    Weird if there's an incompatibility in that version and bummer if you go through this effort to add Python 3 and leave the lastest version behind  :)

**Comment by joaonc** [comment]
Added a cpl comments..  thanks for making this PR!

**Comment by kpdyer** [comment]
Thanks for the PR. Overall looks good. It would be great if you could address jasonc@'s comments.

File: .travis.yml (hunk 2 above)
Comment (sergeybe) [reply .travis.yml]: Due to this issue:
https://github.com/travis-ci/travis-ci/issues/9815

Travis CI doesn't support directly Python 3.7 only through matrix option in .travis.yml. I've added it.

File: setup.py (hunk 1 above)
Comment (sergeybe) [reply setup.py]: Thank you! I've changed to `pycryptodome` because it is better.

**Comment by sergeybe** [comment]
Thank you!

Do you going to update this package on PyPI? There is old version: https://pypi.org/project/libffx/

**Pull request merged**